agent/agent-configuration.mdx

```diff
@@ -38,46 +54,40 @@ However, to take Marco to the next level, we need to give him some Tools to be a
 Marco would not be able to conduct market research without access to data sources and analysis tools.  
 Let's make sure he has the right Tools for the job.
 
-## Equip with Tools
-
-<Tip>
- Experiment and feedback have proven that best Agents are created by domain
- experts. Domain experts have the knowledge to design Tools that are best
- tailored to companies' specific needs. Consequently, Agents equipped with 
- custom Tool(s) are more successful in addressing company-specific challenges.
-</Tip>
+## Equip your agent with tools
 
-Tools are ways for Agents to interact with other systems. They can be used to perform actions, retrieve information and more. 
-In Relevance, [Tools that you create](/build-custom-tools/introduction) are the skills with which you equip your Agents.
-This is extremely useful for two reasons:
+You can equip your agent with any of our [in-built tools](#), or with [tools you have created from scratch](#).
 
-1. Agents are not limited to or restricted by the available functionality on the platform; you can expand on any functionality that you desire. 
-2. The best Agents have Tools created for them by domain experts in a specific company and for a specific use case.
+Just like in the Matrix where [Neo instantly learned kung fu](https://www.youtube.com/watch?v=0YhJxJZOWBw) just by downloading a program, agents can learn new superpowers just by equipping them with skills or domain expertise.
```

Comment:
fine for now but ideally we have the first example as something a bit more business focused rather than a fun example. I do like this fun example but probably dont want to set this tone though as first impression

Reply:
Replaced with an FAQ writing tool example :)